spec/features/customers/user_creates_order_spec.rb

```diff
@@ -30,7 +30,7 @@
     expect(page).to have_content('Order Status: Pending')
     within('.cart') do
       rest1.items.each do |item|
-        unless item.price == 'No Price'
+      if item.price.match(/^\d+\.?\d{0,2}$/)
```

Comment:
Use `=~` in places where the `MatchData` returned by `#match` will not be used.<br>Use 2 (not 0) spaces for indentation.